src/views/admin/components/ApplicationSearch/ApplicationSearch.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+	ChangeEvent,
+	Dispatch,
+	SetStateAction,
+	useEffect,
+	useState,
+} from "react";
+import { Container, Form } from "react-bootstrap";
+import { SidebarDisplay, Status } from "views/admin/applications/AdminHome";
+import styles from "./ApplicationSearch.module.scss";
+import SearchResults from "./SearchResults/SearchResults";
+
+interface ApplicationSidebarProps {
+	data: SidebarDisplay[];
+	setUser: Dispatch<SetStateAction<number>>;
+}
+
+function ApplicationSearch({ data, setUser }: ApplicationSidebarProps) {
+	const [displayedApplicants, setDisplayedApplicants] = useState<number[]>([]);
+	const [showAccepted, setShowAccepted] = useState<boolean>(false);
+	const [showRejected, setShowRejected] = useState<boolean>(false);
+	const [showWaitlisted, setShowWaitlisted] = useState<boolean>(false);
+	const [showPending, setShowPending] = useState<boolean>(true);
```

Comment:
Suggestion: instead of having several states for the filter, we could store a list of statuses to show.